### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DannyPeck**
# Admin Portal
Administrators' portal to managing their Organization.

## Features
- Can create a new organization.
- Can create a new user in the organization.
- Can view all users within their organization.
- Can view all organizations.
- Can view all messages sent from within their organization.

### CODE REVIEW

File: public_html/scripts/userList.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$.getJSON( "/users", function( data ) {
+    $("#user-list").append("<tr> <th> UserName </th> <th> Email </th> <th> Job Title </th> </tr>");
+    $.each( data, function( index, item ) {
+        // if your IDE complains about this, ignore it, it is wrong
```
Comment (eburgun) [inline public_html/scripts/userList.js:4]: What is this comment and commented code for? Can we get rid of it?

File: public_html/scripts/messageList.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+$.getJSON( "/messages", function( data ) {
+    $("#message-list").append("<tr> <th> Sender </th> <th> Recipient </th> <th> Content </th> </tr>");
+    $.each( data, function( index, item ) {
+        // if your IDE complains about this, ignore it, it is wrong
```
Comment (eburgun) [inline public_html/scripts/messageList.js:4]: Do we need this comment?

File: public_html/scripts/buildOrgList.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+$.getJSON( "/organization", function( data ) {
+    $("#contact-list").append("<tr> <th> Organization </th> <th> Location </th> <th> Website </th> </tr>");
+    $.each( data, function( index, item ) {
+        // if your IDE complains about this, ignore it, it is wrong
```
Comment (eburgun) [inline public_html/scripts/buildOrgList.js:4]: Do we need this comment?

**Review by eburgun — COMMENTED** [review]
There are a few comments primarily about commented code and a few comments

File: public_html/scripts/buildOrgList.js (hunk 3 above)
Comment (DannyPeck) [reply public_html/scripts/buildOrgList.js:4]: No we do not. Sigh.

**Pull request merged**